### ISSUE TITLE
Error running script that uses flutter library

### ISSUE THREAD

**Issue by icatalud** [flutter/flutter#35608]
When attempting to run a scripts that uses Element and Widgets, but no rendering, the only way I can run it, is using 'flutter test' command, but this does not satisfy my needs, since assertions cannot be disabled.

When I run it with dart command it throws error about not finding dart:ui on many packages, like:

> .../flutter/packages/flutter/lib/src/material/animated_icons.dart:9:8: Error: Not found: 'dart:ui'
> import 'dart:ui' as ui show Paint, Path, Canvas;
> 
> .../flutter/packages/flutter/lib/src/material/animated_icons.dart:10:8: Error: Not found: 'dart:ui'
> import 'dart:ui' show lerpDouble;
> 

Is this a bug?

**Comment by BondarenkoStas** [comment]
May be related to #27826 and #32680

**Comment by jonahwilliams** [comment]
You cannot run code that depends on `dart:ui` even transitively, such as Flutter, on the standalone dart VM.

**Comment by icatalud** [comment]
@jonahwilliams , @BondarenkoStas 

>You cannot run code that depends on dart:ui even transitively, such as Flutter, on the standalone dart VM.

I didn't know if it was possible or not, but I raised the issue because I wondered, shouldn't it be possible somehow? It's particularly useful for running Flutter dependent scripts.

I required this long time ago for running Flutter benchmarks that don't create a UI.

Flutter test is the only way I found to be able to run such scripts. The problem is that tests have assertions on (there is no way to disable them), they are therefore not good for benchmarking. Is there currently a way to run code that depends on dart:ui without using Flutter or Flutter test?
